update.sh

```diff
@@ -9,25 +9,51 @@ if [ ${#versions[@]} -eq 0 ]; then
 fi
 versions=( "${versions[@]%/}" )
 
-packagesBase='http://apt.postgresql.org/pub/repos/apt/dists/jessie-pgdg'
-mainList="$(curl -fsSL "$packagesBase/main/binary-amd64/Packages.bz2" | bunzip2)"
+declare -A debianSuite=(
```

Comment:
So, if/when we get PostgreSQL 11, this will break, won't it?  Is the idea just to force us to explicitly reconsider at that point which base we should use?

(Should probably update from `set -eo pipefail` to `set -Eeuo pipefail` like we've done in several other scripts to help ensure that breakage _does_ occur, and gives a semi-sane error message about an unset variable in `debianSuite`.)